src/components/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,74 @@
+import "./Sidebar.css";
+
+import { useState } from "react";
+
+import { Icon, IconNames } from "..";
+
+interface Item {
+  icon: IconNames;
+  text: string;
+}
+interface Section {
+  heading: string;
+  items: Item[];
+}
+interface SidebarProps {
+  isVisible: boolean;
+  sections: Section[];
+}
+export const Sidebar = ({ ...props }: SidebarProps) => {
+  const { sections } = props;
+  const [isCollapsed, setCollapsed] = useState(false);
+
+  const toggleVisibility = () => {
+    setCollapsed(!isCollapsed);
+  };
+
+  return (
+    <div className="sidebar">
+      <div className={`sidebar-full ${isCollapsed ? "sidebar-collapsed" : ""}`}>
+        <div className="menu-header">
+          <button
+            type="button"
+            className="usa-button usa-button--outline usa-button--inverse"
+            onClick={toggleVisibility}
+          >
+            {!isCollapsed ? "<<" : ">>"}
+          </button>
+        </div>
+
+        {sections.map((section, sectionIndex) => (
+          <div key={sectionIndex} className="menu-item">
+            {!isCollapsed && (
+              <p className="overline section-heading">{section.heading}</p>
+            )}
+            {section.items.map((item, itemIndex) => (
+              <div
+                key={itemIndex}
+                className={`item ${isCollapsed ? "visible" : "hidden"}`}
+              >
+                <ul>
+                  <li>
+                    <Icon className="icon" name={item.icon} />
+                    {!isCollapsed && <span className="text">{item.text}</span>}
+                  </li>
+                </ul>
+              </div>
+            ))}
+          </div>
+        ))}
+
+        <div className="menu-footer">
+          <ul>
+            <li>
+              <span className="icon">
+                <Icon className="icon" name="logout" />
+              </span>
+              <span className="text">Logout</span>
```

Comment:
This logout button isn't aligned like the rest of the buttons.  Ideally, it should be treated the same and have the same style.
<img width="112" alt="image" src="https://github.com/CDCgov/cdc-react/assets/98106992/5daba6ee-32d9-4033-b76d-db0a829973a1">